frontend/src/components/explorer/gemBrowser/Gene.vue

```diff
@@ -1,40 +1,43 @@
 <template>
   <component-layout
-    component-type="gene" :component-name="gene.geneName"
-    :external-dbs="gene.externalDbs" query-component-action="genes/getGeneData"
-    :interaction-partner="true" :viewer-selected-i-d="gene.id"
+    component-type="gene"
+    :component-name="gene.geneName"
+    :external-dbs="gene.externalDbs"
+    query-component-action="genes/getGeneData"
+    :interaction-partner="true"
+    :viewer-selected-i-d="gene.id"
   >
-    <template v-slot:table>
+    <template #table>
```

Comment:
Is that fix done manually (and why) or by the Prettier?

Reply:
This was done to make eslint (`--fix`), but is [now restored](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/797/commits/0cef748835c1044dc72db8e9b0a045017f426234#diff-e885f43c224f5da447be0fb7dc905906bdc760dcdb5b1bc2d3c64268de83bb93R10).
The [corresponding rule is disabled](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/797/commits/d02abb96e4be15e989eacb13de0d23d401b15ea9) for now.